exchanges/orderbook/orderbook_test.go

```diff
@@ -114,62 +114,51 @@ func TestVerify(t *testing.T) {
 
 func TestCalculateTotalBids(t *testing.T) {
 	t.Parallel()
-	base := Book{
-		Pair:        currency.NewBTCUSD(),
-		Bids:        []Level{{Price: 100, Amount: 10}},
-		LastUpdated: time.Now(),
-	}
-
-	a, b := base.TotalBidsAmount()
-	if a != 10 && b != 1000 {
-		t.Fatal("TestCalculateTotalBids expected a = 10 and b = 1000")
-	}
+	b := Book{Pair: currency.NewBTCUSD(), Bids: []Level{{Price: 100, Amount: 10}}, LastUpdated: time.Now()}
+	ac, total := b.TotalBidsAmount()
+	require.Equal(t, 10.0, ac)
+	require.Equal(t, 1000.0, total)
```

Comment:
🚧 Inconsistency with test below, L127
Don't think these need to be `require`, but that means changing L127. Endless 🔁 
```suggestion
	assert.Equal(t, 10.0, ac, "should return amount")
	assert.Equal(t, 1000.0, total, "should return total")
```